RecoPixelVertexing/PixelTriplets/src/CAHitQuadrupletGenerator.cc

```diff
@@ -43,7 +43,10 @@ fitFastCircleChi2Cut(cfg.getParameter<bool>("fitFastCircleChi2Cut")),
 useBendingCorrection(cfg.getParameter<bool>("useBendingCorrection")),
 caThetaCut(cfg.getParameter<double>("CAThetaCut")),
 caPhiCut(cfg.getParameter<double>("CAPhiCut")),
-caHardPtCut(cfg.getParameter<double>("CAHardPtCut"))
+caHardPtCut(cfg.getParameter<double>("CAHardPtCut")),
+layerList(cfg.getParameter<std::vector<std::string>>("layerList")),
+isFastSim(cfg.getParameter<bool>("isFastSim")),
+layerPairs(cfg.getParameter<std::vector<unsigned>>("layerPairs"))
```

Comment:
The 'layerList' and 'isFastSim' parameters are used by the SeedingLayerSetsBuilder which I call from SeedFinderSelector. 'LayerPairs' is used in SeedFinderSelector to form the desired hit doublets for a particular iteration, I'm keeping this parameter so that it is flexible for changes. May be I can think of something better than adding just useless parameters to CA class.